pkg/cookies/cookies_test.go

```diff
@@ -76,4 +79,74 @@ var _ = Describe("Cookie Tests", func() {
 			}),
 		)
 	})
+
+	Context("MakeCookieFromOptions", func() {
+		type MakeCookieFromOptionsTableInput struct {
+			host           string
+			name           string
+			value          string
+			opts           options.Cookie
+			expiration     time.Duration
+			now            time.Time
+			expectedOutput time.Time
+		}
+
+		validName := "_oauth2_proxy"
+		validSecret := "secretthirtytwobytes+abcdefghijk"
+		domains := []string{"www.cookies.test"}
+
+		now := time.Now()
+		var expectedExpires time.Time
+
+		DescribeTable("should return expected results",
```

Comment:
Since we are only checkin the expiry here, should we add that into the title of the test?

Reply:
Absolutely.
I changed the title.